<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="kr.co.ureca.service.SseServiceTest" tests="1" skipped="0" failures="1" errors="0" timestamp="2024-09-30T09:59:18" hostname="igayeon-ui-MacBookAir.local" time="5.471">
  <properties/>
  <testcase name="testEmitterOnCompletion()" classname="kr.co.ureca.service.SseServiceTest" time="5.471">
    <failure message="org.awaitility.core.ConditionTimeoutException: Condition with kr.co.ureca.service.SseServiceTest was not fulfilled within 5 seconds." type="org.awaitility.core.ConditionTimeoutException">org.awaitility.core.ConditionTimeoutException: Condition with kr.co.ureca.service.SseServiceTest was not fulfilled within 5 seconds.
	at app//org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:165)
	at app//org.awaitility.core.CallableCondition.await(CallableCondition.java:78)
	at app//org.awaitility.core.CallableCondition.await(CallableCondition.java:26)
	at app//org.awaitility.core.ConditionFactory.until(ConditionFactory.java:895)
	at app//org.awaitility.core.ConditionFactory.until(ConditionFactory.java:864)
	at app//kr.co.ureca.service.SseServiceTest.testEmitterOnCompletion(ReservationSseTest.java:72)
	at java.base@17.0.12/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base@17.0.12/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base@17.0.12/java.util.ArrayList.forEach(ArrayList.java:1511)
</failure>
  </testcase>
  <system-out><![CDATA[Registering onCompletion callback...
Calling emitter.complete()...
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
